mission_summary/mission_summary.py

```diff
@@ -0,0 +1,209 @@
+import pandas as pd
```

Comment:
should this be a submodule of glidertest? like glidertest/mission_summary

also, try to avoid having a file with the same name as the folder! 

Whoever made us all have to write `from datetime import datetime` has a lot to answer for